app.js

```diff
@@ -47,14 +51,17 @@ app.set('view engine', 'ejs')
 app.use('/api/v1/auth', authRouter)
 app.use('/api/v1/recipes', auth, recipes)
 
-app.post('/googleTicket', (req, res) => {
-  res.redirect('/')
-})
+//app.post('/googleTicket', (req, res) => {
+//  res.redirect('/')
+//})
+
+app.use(express.static('./MyKitchenApp-front/build'))
+
 app.get('/googleTicket', (req, res) => {
   res.render('index', {client_id: process.env.CLIENT_ID})
 })
 
-app.get('/', (req, res) => {
+app.get('/routeTest', (req, res) => {
```

Comment:
is this testing route?

Reply:
Vaidehi, Dahlak added the "/" route just for testing, in particular so that the front end folks could verify that the back end was up.  I don't know that we need it in the long term, but I had to move it from "/" to "/routeTest" because, now that the back end serves up front end pages, the "/" route is the one that brings up the front end.